src/control/client/types_test.go

```diff
@@ -0,0 +1,68 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package client
+
+import (
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/common"
+)
+
+func TestAccessControlList_String(t *testing.T) {
+	for name, tc := range map[string]struct {
+		acl    *AccessControlList
+		expStr string
+	}{
+		"nil": {
```

Comment:
Not actually nil, is it? Looks like your stringer implementation should already handle a nil, though.

Reply:
Oops, good catch. Copy/paste error. Fixing.